### PR TITLE
fix(decline): fix typo

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

add "s" to /application

## Why

typo error

## Issue

#830 

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [ ] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/features/deleteCompany/deleteCompanyApiSlice.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const apiSlice = createApi({
     }),
     declineRegistration: builder.mutation<void, string>({
       query: (applicationId) => ({
-        url: `/api/registration/application/${applicationId}/declineregistration`,
+        url: `/api/registration/applications/${applicationId}/declineRegistration`,
```
Comment (evegufy) [inline src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: why are you changing again to camelcase for declineRegistration, you just changed it: https://github.com/eclipse-tractusx/portal-frontend/pull/831

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/features/deleteCompany/deleteCompanyApiSlice.ts (hunk 1 above)
Comment (manojava-gk) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: Fix mentioned in the task for "s" and not for camel case. So reverted the change. @jjeroch / @Phil91  can anyone confirm this?

Comment (evegufy) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: @jjeroch also mentioned the endpoint in the issue ` /api/registration/applications/{applicationId}/declineregistration`, camelcase is wrong as well, IMO

Comment (manojava-gk) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: But when I spoke to her, she mentioned that fix is to add "s" to the application and not camel case change. Will wait for the confirmation. I can change it from camel to all small case

Comment (oyo) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: @evegufy @manojava-gk the documentation is not 100% correct there as the example in line 161 doesn't match the  path in line 169

https://github.com/eclipse-tractusx/portal-backend/blob/a0b4b0d5729252bdf5b012d4cd592789092b68aa/src/administration/Administration.Service/Controllers/RegistrationController.cs#L161-L169

Comment (Phil91) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: @manojava-gk @oyo the correct url is /declineregistration

see [RegistrationController](https://github.com/eclipse-tractusx/portal-backend/blob/42e73066a27fb5a7343d4b08459f832eb9334c85/src/registration/Registration.Service/Controllers/RegistrationController.cs#L510) as reference

Comment (oyo) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts:49]: @Phil91 yes all lowercase, but still there is still an example in the documentation line 500 that doesn't match the path in line 507 (missing 's' after application)

[RegistrationController.cs#L507](https://github.com/eclipse-tractusx/portal-backend/blob/release/v2.0.0-RC10/src/registration/Registration.Service/Controllers/RegistrationController.cs#L507)

`[Route("applications/{applicationId}/declineregistration")]`

[RegistrationController.cs#L500](https://github.com/eclipse-tractusx/portal-backend/blob/release/v2.0.0-RC10/src/registration/Registration.Service/Controllers/RegistrationController.cs#L500)

```diff
- /// Example: POST: /api/registration/application/{applicationId}/declineregistration
+ /// Example: POST: /api/registration/applications/{applicationId}/declineregistration
```

**Review by oyo — APPROVED** [review]

**Review by oyo — CHANGES_REQUESTED** [review]
`declineRegistration` should be lower case - see discussion

**Review by evegufy — APPROVED** [review]

**Pull request merged**